perception/image_projection_based_fusion/src/utils/utils.cpp

```diff
@@ -13,8 +13,33 @@
 // limitations under the License.
 
 #include "image_projection_based_fusion/utils/utils.hpp"
+
 namespace image_projection_based_fusion
 {
+Eigen::Vector2d calcRawImageProjectedPoint(
+  const image_geometry::PinholeCameraModel & pinhole_camera_model, const cv::Point3d & point3d)
+{
+  cv::Point2d rectified_image_point = pinhole_camera_model.project3dToPixel(point3d);
+
+  cv::Point2d raw_image_point = rectified_image_point;
+
+  try {
```

Comment:
What is this try sentence for?

Reply:
The code for determining whether to unrectify in the library of image_geometry seems to change occasionally. If it can't be done, it is thrown, so I want to be able to catch it.

Reply:
Hmm, understood. It is a difficult decision, but IMHO we should avoid catching this error and just logging warning without killing the node, since it will be difficult for developers to be aware of a breaking change or a bug in `image_geometry` library, increasing a risk of a potential degradation of Autoware. What do you think?

Reply:
@kminoda So, what exactly do you want to do?

Reply:
Please see here https://github.com/autowarefoundation/autoware.universe/pull/7030#discussion_r1604267322

Reply:
@kminoda This time, when d is 0 in the image_geometry class, it is determined that the image is not calibrated, and thus there is no need to unrectify, resulting in an exception being thrown. This determination of being uncalibrated seems to change frequently in the code. Even if an exception is thrown, there is no need to terminate the process since it only means that unrectifying is not possible. Wouldn't it be better to skip this with a try-catch block?

- https://github.com/ros-perception/vision_opencv/blob/humble/image_geometry/src/pinhole_camera_model.cpp#L157-L160

- https://docs.ros.org/en/diamondback/api/image_geometry/html/c++/pinhole__camera__model_8cpp_source.html#l00328

Reply:
I will close this PR as there seems to have been a misunderstanding on my part.